display.go

```diff
@@ -17,40 +17,38 @@ const (
 )
 
 var (
-	DefaultChannel = machine.I2C0
-	DefaultFont    = &proggy.TinySZ8pt7b
-	White          = color.RGBA{255, 255, 255, 255}
+	DefaultFont = &proggy.TinySZ8pt7b
+	White       = color.RGBA{255, 255, 255, 255}
 )
 
-// Display is a wrapper around `ssd1306.Device` for drawing graphics and text to the OLED.
-type Display struct {
+type display struct {
 	ssd1306.Device
```

Comment:
Another exported field on an unexported type which is readily accesible to users.